modular_citadel/code/modules/reagents/reagent container/hypospraymkii.dm

```diff
@@ -112,23 +112,23 @@
 /obj/item/hypospray/mkii/attackby(obj/item/I, mob/living/user)
 	if((istype(I, /obj/item/reagent_containers/glass/bottle/vial) && vial != null))
 		if(!quickload)
-			to_chat(user, "<span class='warning'>\The [src] can not hold more than one vial!</span>")
+			to_chat(user, "<span class='warning'>[src] can not hold more than one vial!</span>")
 			return FALSE
 		unload_hypo(vial, user)
 	if((istype(I, /obj/item/reagent_containers/glass/bottle/vial)))
 		var/obj/item/reagent_containers/glass/bottle/vial/V = I
 		if(!is_type_in_list(V, allowed_containers))
-			to_chat(user, "<span class='notice'>\The [src] doesn't accept this [vial].</span>")
+			to_chat(user, "<span class='notice'>[src] doesn't accept this [vial].</span>")
```

Comment:
How does improper noun parsing work anyway? From what I've seen, \the and \The are used when an improper noun needs to be preceded with "the", and is only applied when the name of the object isn't capitalized (or if it must be capitalized, it isn't preceded with \improper).

Would this line possibly read as "The hypospray doesn't accept this the hypovial." ?

Reply:
this is also why I didn't want [vial] to be named. :v